ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -1436,18 +1436,18 @@ void connect_memory_and_alias(ast_node_t* hb_instance, char* instance_name_prefi
             char* alias_name;
 
             /* this will crash because of an oassert. we keep it but we work around the problem here */
-            if (hb_instance_var_node->type == RANGE_REF) {
-                full_name = make_full_ref_name(instance_name_prefix, NULL, NULL, NULL, -1);
-            } else {
-                name_of_hb_input = get_name_of_pin_at_bit(hb_instance_var_node, -1, instance_name_prefix, local_ref);
-                full_name = make_full_ref_name(instance_name_prefix, NULL, NULL, name_of_hb_input, -1);
-                vtr::free(name_of_hb_input);
-            }
+            // if (hb_instance_var_node->type == RANGE_REF) {
```

Comment:
can you uncomment for now, It should be tested in a separate PR with tests and all (sorry forgot to remove my changes.)